examples/dt/uni04delta/edpm-pre-ceph/nodeset/values.yaml

```diff
@@ -0,0 +1,152 @@
+              primary: true
+          {% for network in nodeset_networks %}
+            - type: vlan
+              mtu: {{ lookup('vars', networks_lower[network] ~ '_mtu') }}
+              vlan_id: {{ lookup('vars', networks_lower[network] ~ '_vlan_id') }}
+              addresses:
+              - ip_netmask:
+                  {{ lookup('vars', networks_lower[network] ~ '_ip') }}/{{ lookup('vars', networks_lower[network] ~ '_cidr') }}
+              routes: {{ lookup('vars', networks_lower[network] ~ '_host_routes') }}
+          {% endfor %}
+        edpm_nodes_validation_validate_controllers_icmp: false
+        edpm_nodes_validation_validate_gateway_icmp: false
+        edpm_selinux_mode: enforcing
+        edpm_sshd_allowed_ranges:
+          - 192.168.122.0/24
+        edpm_sshd_configure_firewall: true
+        enable_debug: false
+        gather_facts: false
+        neutron_physical_bridge_name: br-ex
+        neutron_public_interface_name: eth0
+        edpm_ceph_hci_pre_enabled_services:
+          - ceph_mon
+          - ceph_mgr
+          - ceph_osd
+          - ceph_rgw
+          - ceph_nfs
+          - ceph_rgw_frontend
+          - ceph_nfs_frontend
+        storage_mtu: 9000
+        storage_mgmt_mtu: 9000
+        storage_mgmt_vlan_id: 23
+        storage_mgmt_cidr: "24"
+        storage_mgmt_host_routes: []
+
+    networks:
+      - defaultRoute: true
+        name: ctlplane
+        subnetName: subnet1
+      - name: internalapi
+        subnetName: subnet1
+      - name: storage
+        subnetName: subnet1
+      - name: tenant
+        subnetName: subnet1
+      - name: storagemgmt
+        subnetName: subnet1
+
+    nodes:
+      edpm-ceph-0:
+        ansible:
+          ansibleHost: 192.168.122.105
+        hostName: edpm-ceph-0
+        networks:
+          - defaultRoute: true
+            fixedIP: 192.168.122.105
+            name: ctlplane
+            subnetName: subnet1
+          - name: storage
+            subnetName: subnet1
+          - name: storageMgmt
+            subnetName: subnet1
+      edpm-ceph-1:
+        ansible:
+          ansibleHost: 192.168.122.106
+        hostName: edpm-ceph-1
+        networks:
+          - defaultRoute: true
+            fixedIP: 192.168.122.106
+            name: ctlplane
+            subnetName: subnet1
+          - name: storage
+            subnetName: subnet1
+          - name: storageMgmt
+            subnetName: subnet1
+      edpm-ceph-2:
+        ansible:
+          ansibleHost: 192.168.122.107
+        hostName: edpm-ceph-2
+        networks:
+          - defaultRoute: true
+            fixedIP: 192.168.122.107
+            name: ctlplane
+            subnetName: subnet1
+          - name: storage
+            subnetName: subnet1
+          - name: storageMgmt
+            subnetName: subnet1
+
+    services:
+      - bootstrap
+      - configure-network
+      - validate-network
+      - install-os
+      - ceph-hci-pre
```

Comment:
good. that opens the firewall for the ceph services the edpm node will be hosting.